### DISCUSSION TITLE
el kernel desaparece x(

### DISCUSSION THREAD

**Discussion by FranciscoSantos312** [LaunchX-InnovaccionVirtual/CursoIntroPython#243] [Q&A]
hola hace rato tengo este problema, cuando corro el programa no termina y no da error, entonces cierro el VSC y lo vuelvo abrir, pero después ya no me deja seleccionar el kernel para usar python y no me sale el recuadro al costado para correr el programa, tengo dos versiones la 3.9 y la 3.10 la mas reciente, la vez pasada se soluciono cuando lo cerré y lo ejecute como administrador pero esta vez ya no funciono, y no conozco muy bien VSC primera vez que lo uso Xd

![kernel](https://user-images.githubusercontent.com/99147667/153944592-a0993777-1abc-4944-9e4c-440ce9b97299.png)

**Comment by AlejandroSBorges** [comment]
Hola.
Puedes intentar usando "CTRL + Shift + P" para abrir los comandos y ahí buscas el de "Seleccionar Kernel"
